<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="mFrm3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="mLblEscuela">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Escuela</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mCmbBxEscuela"/>
         </item>
         <item>
          <widget class="QToolButton" name="mTlBtnFijarEscuela">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resources/icons32.qrc">
             <normaloff>:/icons32/icons/32/media-playback-start.png</normaloff>:/icons32/icons/32/media-playback-start.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabPosition">
         <enum>QTabWidget::South</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Informaci√≥n General</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QSplitter" name="mSplttrSplitter01">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="childrenCollapsible">
             <bool>true</bool>
            </property>
            <widget class="QFrame" name="mFrm1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>150</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="mLblNivelEducativo">
                <property name="text">
                 <string>Nivel</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="mCmbBxNivelEducativo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="mLblGradoEscolar">
                <property name="text">
                 <string>Grado</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="mCmbBxGradoEscolar">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="mLblGrupo">
                <property name="text">
                 <string>Grupo</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="mCmbBxGrupo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QToolButton" name="mTlBtnBuscar">
                <property name="minimumSize">
                 <size>
                  <width>45</width>
                  <height>45</height>
                 </size>
                </property>
                <property name="text">
                 <string>Buscar</string>
                </property>
                <property name="icon">
                 <iconset resource="resources/icons32.qrc">
                  <normaloff>:/icons32/icons/32/edit-find.png</normaloff>:/icons32/icons/32/edit-find.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QFrame" name="mFrm2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QFrame" name="mFrn4">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QToolButton" name="mTlBtnRecargar">
                   <property name="text">
                    <string>Recargar</string>
                   </property>
                   <property name="icon">
                    <iconset resource="resources/icons32.qrc">
                     <normaloff>:/icons32/icons/32/view-refresh.png</normaloff>:/icons32/icons/32/view-refresh.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>22</width>
                     <height>22</height>
                    </size>
                   </property>
                   <property name="toolButtonStyle">
                    <enum>Qt::ToolButtonTextBesideIcon</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="mTlBtnNuevo">
                   <property name="text">
                    <string>Nuevo</string>
                   </property>
                   <property name="icon">
                    <iconset resource="resources/icons32.qrc">
                     <normaloff>:/icons32/icons/32/document-new.png</normaloff>:/icons32/icons/32/document-new.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>22</width>
                     <height>22</height>
                    </size>
                   </property>
                   <property name="toolButtonStyle">
                    <enum>Qt::ToolButtonTextBesideIcon</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="mTlBtnModificar">
                   <property name="text">
                    <string>Modificar</string>
                   </property>
                   <property name="icon">
                    <iconset resource="resources/icons32.qrc">
                     <normaloff>:/icons32/icons/32/accessories-text-editor.png</normaloff>:/icons32/icons/32/accessories-text-editor.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>22</width>
                     <height>22</height>
                    </size>
                   </property>
                   <property name="toolButtonStyle">
                    <enum>Qt::ToolButtonTextBesideIcon</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>316</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QToolButton" name="mTlBtnBorrar">
                   <property name="text">
                    <string>Borrar</string>
                   </property>
                   <property name="icon">
                    <iconset resource="resources/icons32.qrc">
                     <normaloff>:/icons32/icons/32/edit-delete.png</normaloff>:/icons32/icons/32/edit-delete.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>22</width>
                     <height>22</height>
                    </size>
                   </property>
                   <property name="toolButtonStyle">
                    <enum>Qt::ToolButtonTextBesideIcon</enum>
                   </property>
                   <property name="arrowType">
                    <enum>Qt::NoArrow</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QTableView" name="mTblVwTablaGeneral"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Consulta Por Persona</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>57</width>
         <height>370</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Page 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>57</width>
         <height>370</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Page 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mMnBrBarraMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionA_brir"/>
   </widget>
   <addaction name="menu_Archivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionA_brir">
   <property name="text">
    <string>A&amp;brir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/icons32.qrc"/>
 </resources>
 <connections/>
</ui>
